features/storage/add_update_delete_volume.feature

```diff
@@ -14,7 +14,7 @@ Feature: Add, update remove volume to rc/dc and --overwrite option
       | name         | mydb                       |
     Then the step should succeed
     And a pod becomes ready with labels:
-      | app=mydb |
+      | deploymentconfig=mydb |
```

Comment:
@pruan-rht Do you test this with an 4.4 version oc client?

Reply:
This was a regression against 4.3.0